src/gui/MainWindow.cpp

```diff
@@ -822,11 +824,6 @@ void MainWindow::showGroupContextMenu(const QPoint& globalPos)
     m_ui->menuGroups->popup(globalPos);
 }
 
-void MainWindow::saveToolbarState(bool value)
-{
-    config()->set("ShowToolbar", value);
```

Comment:
@weslly can we also remove that from the Config?

Reply:
@louib good catch! fixed